subsys/zigbee/osif/zb_nrf_serial.c

```diff
@@ -28,10 +30,9 @@ void zb_osif_async_serial_wake_up(void);
 void zb_osif_serial_init(void)
 {
 #if defined(CONFIG_ZBOSS_TRACE_BINARY_LOGGING) && defined(CONFIG_ZIGBEE_HAVE_ASYNC_SERIAL)
-	/* Assert that Serial Logger and Async Serial don't use the same serial device. */
-	int ret = strcmp(CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME, CONFIG_ZIGBEE_UART_DEVICE_NAME);
-
-	__ASSERT(ret, "The same serial device used for serial logger and async serial!");
+	__ASSERT(DEVICE_DT_GET(DT_CHOSEN(ncs_zigbee_uart)) !=
```

Comment:
I think we can safely remove `#include <string.h>` as no longer needed here

Reply:
done